protos/farm_ng/core/pose.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) farm-ng, inc.
+//
+// Licensed under the Amiga Development Kit License (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://github.com/farm-ng/amiga-dev-kit/blob/main/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "farm_ng/core/quaternion.proto";
+import "farm_ng/core/vector.proto";
+import "farm_ng/core/timestamp.proto";
+
+package farm_ng.core.proto;
+
+// A position and orientation for an object or reference frame in 3D space
+message Pose3 {
+  Timestamp timestamp = 1;
+  // Translation from origin frame 
+  Vector3 translation = 2;
+  // Quaternion denoting orientation
+  Quaternion rotation = 3;
+
+  string frame_id = 4;
```

Comment:
i assume this is the source frame ? missing also a key describing the destination frame of the transformation for this pose